File: sdk/typescript/examples/09_usage.mjs
// 09_usage:用量查询。
import { client } from "./00_helpers.mjs";
const combee = client();
const cell = await combee.cells.create();
await cell.kv.set("x", "1");
await new Promise((r) => setTimeout(r, 6000)); // 等 usage flush
console.log("summary:", await combee.usage.summary());
console.log("cell usage:", await combee.usage.cell(cell.id));

File: sdk/python/examples/09_usage.py
"""09_usage:用量查询。"""
import time
from helpers import client

combee = client()
cell = combee.cells.create()
cell.kv.set("x", "1")
time.sleep(6)  # 等 usage flush
print("summary:", combee.usage.summary())
print("cell usage:", combee.usage.cell(cell.id))
